doc/admin/deployment.md

```diff
@@ -861,7 +862,7 @@ incomplete integration of certificate support as of the 0.6 release.
 To start the DAOS Agent from the command line, run:
 
 ```bash
-$ daos_agent -i
+$ daos_agent -i &
```

Comment:
i think we need a config file now, no?

Reply:
DAOS-4169 should fix the default config file.  But I can add a config to command line